src/core/thread/key_manager.cpp

```diff
@@ -535,7 +535,7 @@ void KeyManager::GetNetworkKey(NetworkKey &aNetworkKey) const
 void KeyManager::GetPskc(Pskc &aPskc) const
 {
 #if OPENTHREAD_CONFIG_PLATFORM_KEY_REFERENCES_ENABLE
-    if (Crypto::Storage::IsKeyRefValid(mPskcRef))
```

Comment:
`IsKeyRefValid()` is also used in `SetPskc()`. Do we need to update that as well?

Reply:
Similar situation in `KeyManager::SetNetworkKey()`?